apps/code-review/src/app/pages/overview/overview.component.html

```diff
@@ -1,9 +1,13 @@
+<ng-template #isLoader>
```

Comment:
```suggestion
<ng-template #loaderTemplate>
```

Також, давай цей ng-template мувнимо в кінець файлу